src/content/docs/nrql/nrql-references/nrql-math-using-select.mdx

```diff
@@ -46,19 +46,15 @@ SELECT average(duration-databaseDuration) FROM Transaction
 
 ## Use advanced math operators with `SELECT` [#advancedmath]
 
-NRQL also includes some advanced mathematical capabilities that can be used for complex calculations. This is helpful if you want to process data to display it more effectively in the UI, or make statistical or psychometric calculations on queried results in a single step.
+NRQL also includes some advanced mathematical capabilities that can be used for complex calculations. This is helpful if you want to process data to display it more effectively in the UI, or make calculations on queried results in a single step.
```

Comment:
Given that we use capabilities to refer to our top-level features, maybe we want to use something else? Formulas, maybe?